test/test_integration_management.cxx

```diff
@@ -1115,12 +1115,20 @@ TEST_CASE("integration: query index management", "[integration]")
             REQUIRE(index->state == "deferred");
         }
 
+        SECTION("public API")
         {
             auto manager = couchbase::cluster(integration.cluster).query_indexes();
             auto ctx = manager.build_deferred_indexes(integration.ctx.bucket, {}).get();
             REQUIRE_SUCCESS(ctx.ec());
         }
 
+        SECTION("core API")
```

Comment:
I think it might be better to just copy the build test and instead of using the manager to build use the request. The way this test is means there aren't any any indexes to build b/c they have previously been built, right?

Reply:
nested SECTION works like multiplication of the cases

https://github.com/catchorg/Catch2/blob/3644b4135d2f4ec37fe6858dfc03b6273b0948ef/docs/tutorial.md#test-cases-and-sections

for each SECTION the test restarted from the beginning

Reply:
Ahhh, okay!  Great 😄